### PR TITLE
Fix cursor loop for template-lint fixer logic #121

### PR DESCRIPTION
**Pull request by lifeart**
https://github.com/lifeart/ember-language-server/pull/122
https://github.com/lifeart/ember-language-server/issues/121

### CODE REVIEW

File: test/bultin-addons/core/template-linter-test.ts [hunk 1]
```diff
@@ -73,4 +74,46 @@ describe('ProjectTemplateLinter', () => {
...
+    const result = await instance.onCodeAction('', params);
+
+    expect(result).toStrictEqual([
+      {
+        edit: {
+          changes: {
+            'layout.hbs': [
+              {
+                range: { start: { line: 0, character: 0 }, end: { line: 2, character: 6 } },
+                newText: '<div class="md:flex p-2 bg-white rounded-lg">\n  <h3 class="mb-4">Hello</h3>\n</div>',
```
Comment (NullVoxPopuli) [inline test/bultin-addons/core/template-linter-test.ts:109]: somewhat unrelated, but is there a rule for 20+ tailwind classes? having all of those on one line is obnoxious, and it'd be great if there was an auto-fixer to multi-line them

**Comment by NullVoxPopuli** [comment]
:+1: I'm a fan of how the test is 10x longer than the fix :D

**Review by NullVoxPopuli — APPROVED** [review]

File: test/bultin-addons/core/template-linter-test.ts (hunk 1 above)
Comment (lifeart) [reply test/bultin-addons/core/template-linter-test.ts:109]: cc @MichalBryxi

Comment (MichalBryxi) [reply test/bultin-addons/core/template-linter-test.ts:109]: I believe that would belong to the template-lint plugin then? https://gitlab.com/michal.bryxi/ember-template-lint-plugin-tailwindcss/-/issues/4

**Pull request merged**